src/app/models/test-type.model.ts

```diff
@@ -15,4 +15,8 @@ export interface TestType {
   prohibitionIssued: boolean;
 
   testResult: string;
+
+  seatbeltInstallationCheckDate: boolean;
```

Comment:
Should seatbeltInstallationCheckDate have 'Date' at the end? Sounds wrong when it's a boolean.

Reply:
I was also confused, that is the way it's laid out on the DB according to the spreadsheet I was given, it's a boolean called seatbeltInstallationCheckDate, it shows whether or not the seatbelt checks were carried out during this test or not.

Reply:
Sounds like it is a boolean then, just badly named.